Multi-Relay functionality

Ishpishxe opened the pull request:
Enables multiple data sourses to connect to the relay with the same port

Ishpishxe committed:
Multi-Relay functional

Enables multiple data sourses to connect to the relay with the same port

Ishpishxe committed:
This is needed

daniel-centore reviewed (changes requested):
Overall looks good. Formatting is crap. Please reduce blank lines (e.g. never after a { or before a }, never more than 1, etc) and fix indentation levels. Take advantage of eclipse's auto formatter and see the rest of the code for examples of acceptable formatting.

Ishpishxe committed:
Formating

Formating should be fixed and consistant across the board. A few wraped lines got concatenated but it should be the only unintended change.

daniel-centore reviewed (approved):
lgtm
